File: 02_ProgrammingFundamentals/_10_TextProcessing/_02_Exercise/_04_CaesarCipher.java
package _02_ProgrammingFundamentals._10_TextProcessing._02_Exercise;

import java.util.Scanner;

public class _04_CaesarCipher {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        String console = scan.nextLine();

        // Encrypt the message
        StringBuilder encrypt = new StringBuilder();
        for (int i = 0; i < console.length(); i++) {
            char symbol = (char) (console.charAt(i) + 3);

            encrypt.append(Character.valueOf(symbol));
        }

        // Print result
        System.out.println(encrypt);
    }
}

File: 24_C#Fundamentals/_08_TextProcessing/_02_Exercise/_04_CaesarCipher/_04_CaesarCipher.cs
using System;
using System.Text;

public class _04_CaesarCipher
{
    public static void Main(string[] args)
    {
        // Read input value
        string console = Console.ReadLine();

        // Add variable
        StringBuilder encrypt = new StringBuilder();

        // Encrypt the message
        for (int i = 0; i < console.Length; i++)
        {
            char symbol = (char)(console[i] + 3);

            encrypt.Append(symbol);
        }

        // Print result
        Console.WriteLine(encrypt);
    }
}
